0_intro_to_jax_for_ml.ipynb

```diff
@@ -0,0 +1,2081 @@
+{
```

Comment:
<p>I'd avoid the list comprehension here (since some folks might not be familiar with them) and rather manually construct the list using a good old fashioned for loop.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2022/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='91'/>

Reply:
<p>Updated. </p>